var/spack/repos/builtin/packages/fastmath/package.py

```diff
@@ -0,0 +1,58 @@
+##############################################################################
```

Comment:
I didn't notice this the first time, but you probably shouldn't add the tarball to Spack. Users will have to download it manually and put it in the current directory or in a mirror.